frida_constants/include/frida_constants/manipulation_constants_cpp.hpp

```diff
@@ -70,4 +70,6 @@ conststr DEBUG_POSE_GOAL_TOPIC = "/manipulation/debug_pose_goal";
 conststr GET_JOINT_SERVICE = "/manipulation/get_joints";
 conststr TOGGLE_SERVO_SERVICE = "/manipulation/toggle_servo";
 
+constexpr float SQ_SMALL_CLOUD_RADIUS = 1.5f * 1.5f;
+constexpr float SQ_MEDIUM_CLOUD_RADIUS = 2.5f * 2.5f;
```

Comment:
please move these back to the down sample file and make them params of the node

Reply:
And keep them as it radius value, you could later make a constant of the squared of the radius that holds its value. but its important to keep the semantic value here